### ISSUE TITLE
Simplify output option names

### ISSUE THREAD

**Issue by JonoYang** [nexB/scancode-toolkit#959]
Some output formats are prefixed with "output" while others are not. For the sake of consistency and simplicity, "output" should be dropped as a prefix to these output options.

**Comment by yash-nisar** [comment]
Do you mean removing the output prefix from the `./scancode --help` command output as given below ? @JonoYang @pombredanne 

```
  output formats:
    --json FILE             Write scan output as compact JSON to FILE.
    --json-pp FILE          Write scan output as pretty-printed JSON to FILE.
    --json-lines FILE       Write scan output as JSON Lines to FILE.
    --output-csv FILE       Write scan output as CSV to FILE.
    --output-html FILE      Write scan output as HTML to FILE.
    --output-custom FILE    Write scan output to FILE formatted with the custom
                            Jinja template file.
    --custom-template FILE  Use this Jinja template FILE as a custom template.
    --output-html-app FILE  Write scan output as a mini HTML application to
                            FILE.
    --output-spdx-rdf FILE  Write scan output as SPDX RDF to FILE.
    --output-spdx-tv FILE   Write scan output as SPDX Tag/Value to FILE.
```

**Comment by pombredanne** [comment]
Closed as dupe of #789

**Comment by pombredanne** [comment]
@yash-nisar yes mostly. Please continue the discussion on #789
